### PR TITLE
Small improvements in installation section [ECR-3854]

### PR DESCRIPTION
**Pull request by vitvakatu**
Question: should we add `System requirements` section? If so, requirements for _what purposes_ should it describe? The requirements for Exonum node server and for developer environment are obviously quite different.

### CODE REVIEW

File: src/get-started/java-binding.md [hunk 1]
```diff
@@ -21,19 +21,23 @@ To run a node with your Java service you need to use Exonum Java application.
...
-- [Manual installation](#manual-installation) — available for Mac OS and Linux
+- [Manual installation](#manual-installation) — available for Mac OS and Linux,
+  _recommended for Linux users_
 - [Homebrew package](#homebrew-package) — available for Mac OS only,
-  recommended for Mac users
+  _recommended for Mac users_
 - [Build from source](#build-from-source) — available for Mac OS and Linux
 
+Windows is not supported by Java Binding at the moment, consider using
+[WSL] or [Docker][docker-for-windows].
```
Comment (bullet-tooth) [inline src/get-started/java-binding.md]: WSL link is broken

File: src/get-started/java-binding.md [hunk 2]
```diff
@@ -21,19 +21,23 @@ To run a node with your Java service you need to use Exonum Java application.
 
 There are several installation options:
 
-- [Manual installation](#manual-installation) — available for Mac OS and Linux
+- [Manual installation](#manual-installation) — available for Mac OS and Linux,
+  _recommended for Linux users_
```
Comment (bullet-tooth) [inline src/get-started/java-binding.md]: no apt anymore for Linux users?

Comment (dmitry-timofeev) [reply src/get-started/java-binding.md]: We never actually had it.

File: src/get-started/java-binding.md (hunk 1 above)
Comment (vitvakatu) [reply src/get-started/java-binding.md]: Not really, see [rendered](https://github.com/exonum/exonum-doc/blob/facf574152438fa68273175336bde76691288240/src/get-started/java-binding.md)

**Comment by dmitry-timofeev** [comment]
Regarding the system requirements, we could possibly separate them by the tool/use-case, but likely not with the current documentation structure (single document). BTW, do we say to install JDK?

File: src/get-started/java-binding.md [hunk 3]
```diff
@@ -40,28 +40,28 @@ deployment.
...
-    ```
+2. Install [Libsodium][libsodium] as the necessary runtime dependency.
+
+    !!! note
+        Exonum Java is built with Libsodium 23, which means it will not work
+        on some older Linux distributions, like Ubuntu 16.04. Libsodium 23 is
+        available in Ubuntu 18.04 or can be installed from a custom PPA.
+    
+    ??? example "Linux (Ubuntu)"
+        ```bash
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: Would the following work:
```
\```bash tab="Linux (Ubuntu)"

\```bash tab="macOS"
```

https://squidfunk.github.io/mkdocs-material/extensions/codehilite/#grouping-code-blocks

Comment (vitvakatu) [reply src/get-started/java-binding.md]: Wow, cool. It actually works, I checked locally

Comment (vitvakatu) [reply src/get-started/java-binding.md]: изображение

File: src/get-started/java-binding.md [hunk 4]
```diff
@@ -96,7 +94,9 @@ follow the instructions in [Contribution Guide][how-to-build].
     export PATH="$PATH:$EXONUM_HOME/bin"
     ```
 
-2. Install [Maven 3][maven-install] which is essential for developing and building
+2. Install [latest JDK][jdk].
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: Nit: the latest

**Comment by dmitry-timofeev** [comment]
There are conflicts, need to resolve

**Pull request merged**